src/common.rs

```diff
@@ -11,6 +12,9 @@ pub trait AsU32 {
 cfg_if::cfg_if! {
     if #[cfg(any(windows, target_os = "unknown", target_arch = "wasm32"))] {
         /// Process id.
+        /// Can be used as an integer type by simple casting
```

Comment:
So two things here: first, I would write the comment as is:

``````
/// Process id.
///
/// Can be used as an integer type by simple casting. For example:
///
/// ```
/// use sysinfo::AsU32;
///
/// let p: Pid = 0;
/// let i: u32 = p.as_u32();
/// ```
``````

Second, please move both comments into a markdown file like it's done [here](https://github.com/GuillaumeGomez/sysinfo/blob/4f57c32d7bab81b3e67978d60d10e6381545333b/src/linux/system.rs#L100) and put the markdown file in the `md_doc`.